src/fu-util.c

```diff
@@ -2500,6 +2578,16 @@ main (int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+	/* parse filter flags */
+	if (filter != NULL) {
+		if (!fu_util_parse_filter_flags (priv, filter, &error)) {
+			/* TRANSLATORS: the user didn't read the man page */
```

Comment:
Given the man page is generated from `--help`, probably do need to be fairly verbose in the `--help` output about syntax.